kotlin/src/main/kotlin/io/logto/sdk/core/constant/QueryKey.kt

```diff
@@ -0,0 +1,8 @@
+package io.logto.sdk.core.constant
+
+object QueryKey {
```

Comment:
Feel like a little bit too general. Especially under a shared constant file directly under the core.

Reply:
I'm ok with it.
We will have `QueryKey`, `GrantType`, and other constants here. All of them are used in the core.